app/models/user.rb

```diff
@@ -1,17 +1,5 @@
 class User < ActiveRecord::Base
 
-  module Overridable
-
-    def price_groups
```

Comment:
We should figure out a plan for how we're going to merge this into DC because it overrides this module/method. At least until we change their default behavior.